ee/clickhouse/views/events.py

```diff
@@ -52,7 +52,10 @@ def _query_events_list(self, filter: Filter, team: Team, request: Request, long_
         prop_filters, prop_filter_params = parse_prop_clauses(filter.properties, team.pk)
 
         if request.GET.get("action_id"):
-            action = Action.objects.get(pk=request.GET["action_id"])
+            try:
+                action = Action.objects.get(pk=request.GET["action_id"])
+            except:
```

Comment:
Hmm.

Should we handle this case in our code? How can this happen in a legit case? User deletes action in one tab and filters in the other?

If we catch this, this should be `except Action.DoesNotExist:`

Reply:
oh yeah that's the exception.

I was thinking the same thing on that error. seems incredibly odd case but i just propsed this so that it would atleast not crash

Reply:
Let's go with this, though trying to apply the pattern in every endpoint might end up being bit too defensive. Will see